src/models/searchservice/response/DisplayableFacet.ts

```diff
@@ -1,15 +1,11 @@
 import { Facet, FacetOption } from '../request/Facet';
-import { Matcher } from '../common/Matcher';
-import { NumberRangeValue } from '../common/NumberRangeValue';
 
 /**
  * A {@link Facet} which contains extra fields meant to be displayed to the end user.
  *
  * @public
  */
 export interface DisplayableFacet extends Facet {
-  /** {@inheritDoc Facet.fieldId} */
-  fieldId: string,
```

Comment:
Is this now in `Facet`?

Reply:
yes, this property was redundant even before the rework because it's the same as the property in `Facet`. there was no reason to specify it again after extending `Facet`